Bottom: 7eb461b0de84e2bf33f77429094befa17286555d
Top:    d2ac2a2bf59a82f88d383617ef3984a1ef2db56e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-11 14:56:18 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
